packages/datadog-plugin-bluebird/test/index.spec.js

```diff
@@ -14,3 +14,79 @@ assertPromise('bluebird', bluebird => {
 
   return bluebird.getNewLibraryCopy()
 })
+
+describe('bluebird.getNewLibraryCopy', () => {
+  let bluebird
+  const agent = require('../../dd-trace/test/plugins/agent')
+  const plugin = require('../src')
+  const unpatch = plugin[0]['unpatch']
+  const mockIntegrations = {
+    unwrap: sinon.spy()
+  }
+
+  wrapIt()
+
+  describe('library copies to unpatch', () => {
+    withVersions(plugin, 'bluebird', version => {
+      beforeEach(() => {
+        agent.load(plugin, 'bluebird')
+      })
+
+      afterEach(() => {
+        return agent.close()
+      })
+
+      describe('without configuration', () => {
+        beforeEach(() => {
+          bluebird = require(`../../../versions/bluebird@${version}`).get()
+        })
+
+        describe('patching', () => {
```

Comment:
For testing purposes, I think we're less concerned with the implementation details (except that they're not hanging around after unpatch). The previously added test already ensures that it's been patched. The `'unpatching'` tests ensure that we've properly unpatched, so we're good with that I think.

Reply:
k makes sense, as long as the hidden property logic is specified in the unpatching section i think that's fine, updated this `patching` spec removed